### ISSUE TITLE
TypeError: Cannot read property 'removed' of null

### ISSUE THREAD

**Issue by buschtoens** [ember-cli/babel-plugin-filter-imports#30]
Using v1.1.0 through [`@ember-decorators/argument`](https://github.com/ember-decorators/argument) I get the following error for [`ember-radio-button/addon/types.js`](https://github.com/buschtoens/ember-radio-bar/blob/eed16d142fbe49b195c92d60db668e8a8c09f758/addon/types.js) when building for production:

```
The Broccoli Plugin: [BroccoliMergeTrees: Addon#compileAddon(ember-radio-bar) ] failed with:
TypeError: ember-radio-bar/types.js: Cannot read property 'removed' of null
    at /home/jan/github/ember-radio-bar/node_modules/babel-plugin-filter-imports/lib/removeReferences.js:36:16
    at arrayEach (/home/jan/github/ember-radio-bar/node_modules/lodash/lodash.js:537:11)
    at Function.forEach (/home/jan/github/ember-radio-bar/node_modules/lodash/lodash.js:9359:14)
    at removeReferences (/home/jan/github/ember-radio-bar/node_modules/babel-plugin-filter-imports/lib/removeReferences.js:33:20)
    at /home/jan/github/ember-radio-bar/node_modules/babel-plugin-filter-imports/lib/removeReferences.js:45:41
    at arrayEach (/home/jan/github/ember-radio-bar/node_modules/lodash/lodash.js:537:11)
    at Function.forEach (/home/jan/github/ember-radio-bar/node_modules/lodash/lodash.js:9359:14)
    at removeReferences (/home/jan/github/ember-radio-bar/node_modules/babel-plugin-filter-imports/lib/removeReferences.js:33:20)
    at /home/jan/github/ember-radio-bar/node_modules/babel-plugin-filter-imports/lib/index.js:51:49
    at arrayEach (/home/jan/github/ember-radio-bar/node_modules/lodash/lodash.js:537:11)

The broccoli plugin was instantiated at: 
    at BroccoliMergeTrees.Plugin (/home/jan/github/ember-radio-bar/node_modules/broccoli-plugin/index.js:7:31)
    at new BroccoliMergeTrees (/home/jan/github/ember-radio-bar/node_modules/broccoli-merge-trees/index.js:16:10)
    at Function.BroccoliMergeTrees [as _upstreamMergeTrees] (/home/jan/github/ember-radio-bar/node_modules/broccoli-merge-trees/index.js:10:53)
    at mergeTrees (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/broccoli/merge-trees.js:85:33)
    at Class.compileAddon (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/models/addon.js:1092:12)
    at Class.treeForAddon (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/models/addon.js:746:26)
    at Class._treeFor (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/models/addon.js:557:33)
    at Class.treeFor (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/models/addon.js:517:21)
    at project.addons.reduce (/home/jan/github/ember-radio-bar/node_modules/ember-cli/lib/broccoli/ember-app.js:620:25)
    at Array.reduce (<anonymous>)
```

/cc @pzuraq

**Comment by layershifter** [comment]
Released in `babel-plugin-filter-imports@1.1.1`.

Thanks for report 👍 Please check it now.

**Comment by buschtoens** [comment]
Works perfectly. Thank you very much!
